utils/fork_star_desc.js

```diff
@@ -0,0 +1,10 @@
+const axios = require('axios')
```

Comment:
May you don't need Axios as well. Can you try with a plain simple fetch for removing 3rd parties?